packages/rancher-pushprox/charts/values.yaml

```diff
@@ -16,6 +16,8 @@ global:
   cattle:
     systemDefaultRegistry: ""
 
+namespaceOverride: ""
```

Comment:
This new variable is added but not used in the monitoring chart in this PR.

Reply:
Need to add documentation to the README.md for this variable and the others.

Reply:
changes are made based on your comments.